onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -2573,6 +2573,153 @@ ONNX_MS_OPERATOR_SET_SCHEMA(CropAndResize, 1,
         a fixed size = [crop_height, crop_width]. The result is a 4-D tensor [num_boxes, crop_height, crop_width, depth].
         The resizing is corner aligned.)DOC"));
 
+#if !defined(DISABLE_FLOAT8_TYPES)
+#define GEMM_FLOAT8_TYPES \
+  { "tensor(float8e4m3fn)", "tensor(float8e5m2)", "tensor(float16)", "tensor(bfloat16)", "tensor(float)" }
```

Comment:
do we need to support tensor(float16)", "tensor(bfloat16)", "tensor(float)" in this op?

Reply:
Fp8 Gemm is generally fp16 @ fp8 -> fp16 in the case of inference, in training the configuration will be even trickier =)